### DISCUSSION TITLE
Is the project alive and taking contributions?

### DISCUSSION THREAD

**Discussion by gdonval** [wolfi-dev/os#35507] [Q&A]
One month (and running) without an actual review of couple of my PRs, both to add packages here and to improve melange. Your Slack channel is unresponsive. Almost all accepted PRs are automatic version bumps. The project looks dead and on autopilot.

**Comment by xnox** [comment]
It is true that there is a lot more automation; than manual PRs. But there are a few that are done by humans.

I am sorry your prs got burried. As indeed most people review/notice/merge more recent ones most of the time.

Which PRs are outstanding and need attention? Can you please provide links?

I will check the slack situation.

**Reply by gdonval** [reply]
I've submitted:

1. https://github.com/wolfi-dev/os/pull/30266
2. https://github.com/wolfi-dev/os/pull/30306
3. https://github.com/wolfi-dev/os/pull/35256
4. https://github.com/chainguard-dev/melange/pull/1556

Plus a couple others I stopped working on (or require some existing PRs as deps).

I really don't want to sound too negative about it but having to ask for PRs to be looked at as a favour does not bode well. I really like its approach to containers: I find it excellent. But this...
